### DISCUSSION TITLE
Cannot reverify for student benefits

### DISCUSSION THREAD

**Discussion by SammyWhamy** [community/community#51765] [GitHub Education] [answered]
Hello. I've verified 2 years ago that I am a student. My school was unlisted so I manually uploaded documents that prove I am currently enrolled.
I cannot take pictures on my pc as my webcam quality is not high enough to readably capture 2 sheets of paper. Therefore, I took a picture with my phone and uploaded it. I've tried 4 times now but I'm constantly getting rejected with the following:
![image](https://user-images.githubusercontent.com/45788465/229368263-3fe17b59-a985-43e2-b550-e9b7f405e4ac.png)
I already updated my github name to my full name. This is already something I didn't want to do, as I don't need my full name out there for everyone to see, but whatever.

I also tried uploading a screenshot of my microsoft teams and outlook environments, but this was not accepted either.
I totally understand if you guys need additional documents, e.g. exam papers, payment receipts, etc. but you do not allow me to upload more than 1 image, and I'm getting instantly rejected without even being able to give extra info by email. What am I supposed to do? My student pack expires in 7 days, why did this process get so much harder? Last year I was able to do this without a hassle

**Comment by SmashedFrenzy16** [comment]
Hi @SammyWhamy,

Try scanning in (using your phone camera) the academic enrollment proof onto GitHub. Also make sure that you try and not capture anything else in the background. Also try cleaning your phone's camera lens.

If problems persue, don't hesitate to contact your institution. If that does not work, you should try and contact [GitHub support](https://support.github.com/).

Hope this helps!

P.S: Please mark this as the answer to your question if you found this helpful, as I am trying to achieve the Galaxy Bage achievement.

**Comment by subeom7** [comment]
Hi @SammyWhamy,
 
Your verifications should be official documents and should display your expected graduation date. If it does not, ensure that you verify your enrollment using alternative school documents (transcript, proof of enrollment, etc.) that confirm your graduation date

Additionally, link your institution's webpage: In the "Company" or "Website" section of your profile, consider adding a link to your educational institution's webpage to further validate your academic affiliation.

After making these updates, reapply for the GitHub Student Pack. If you continue to face difficulties, consider submitting the request directly to GitHub Support, as they can provide more specific guidance or investigate any issues with your application:

GitHub Education Support: https://support.github.com/contact/education

Hope this helps!

**Reply by samus-aran** [reply]
Hi @subeom7  👋  appreciate the guidance but I want to highlight we would not direct users with these types of inquiries to Education support. We're recently made the announcement we do not process manual reviews for individuals. [Check here ](https://github.com/orgs/community/discussions/49366)for more details on these changes.

**Comment by samus-aran** [comment]
Hi @SammyWhamy 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps. Alternative apply via the mobile interface to utilise the phone's camera.

Hope this helps! Let us know how you get along 👍

**Reply by SammyWhamy** [reply]
I've tried all these steps before but the automated system automatically rejects everything i try.

GitHub support simply closes my ticket and tells me to go back here, but I'm surely not going to upload my private documents on this public forum.

I understand that you guys cannot manually verify every single request, but i think if the automated system fails to verify, you should forward the request to an actual person.

I have about 4 sheets of A4 paper that collectively form the proof that I'm currently enrolled. That simply does not fit into one image, at least not with my camera/webcam quality. GitHub doesn't allow me to upload multiple documents, and most importantly, my documents aren't the most regular school things there are. It's only natural that the automated system has some trouble verifying my name and email, as the documents also list other people's names and emails.

But again, I'm not uploading them here, and i cant upload multiple documents on the application website. And GitHub support just tells me to go here, so what am I supposed to do?

I'm pretty sure i will never be able to satisfy the AI within the automated checks, so i need an actual human to confirm my documents. Can someone help me with that?

**Reply by ahmadhassan7** [reply]
+1 
Getting the same issue.

**Reply by samus-aran** [reply]
Hi @SammyWhamy 

We're sorry to hear of your frustration. As we direct users to utilise the community more and share knowledge on all things GitHub Global Campus sometimes, like this, users require explicit help. But we're here to assist ⭐

Let me try and explain your first rejection reason, please re-read it carefully and adjust your profile and application accordingly.

Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process. 

Look at how your name appears on your evidence as it does on you GitHub account, is there a space, do you have a hyphen, is it translated? If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Let's tackle the first rejection reason first and we can proceed if there are any more.

**Reply by SammyWhamy** [reply]
Hi @samus-aran

Pease re-read my post carefully first.
I have already changed my name on github to match my full legal name which is also on the document. I have changed my primary email to my school email.

The original document is not in english. You suggest I put a translated version next to the original document. I would like to ask you in return, how do you expect me to take a singular picture of **eight** sheets of paper and have them be readable? As well as having to go through the trouble of printing out 4 translated versions

I have already gotten accepted for 2 years before for this exact same school. I am still the same person, at the same school, with the same documents. I have not yet graduated and I am still enrolled.

**Reply by SammyWhamy** [reply]
I understand it's the weekend right now, but I'd just like to mention that for example, my GitHub copilot access is now ending in 3 days and I still have no clue on how to fix this

**Reply by SammyWhamy** [reply]
Seems like I have now lost access to copilot and still haven't received any way to fix this.
To say the least, I am very disappointed in the changes made to the student benefits system.

**Reply by samus-aran** [reply]
Hi @SammyWhamy 

I will ask you to re-read my response. 🙂  I'll highlight in bold where you may need to re-look at your evidence and cross reference your profile details.

Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname? It’s these small but important points to consider when applying that can make a much smoother process.

**Look at how your name appears on your evidence as it does on you GitHub account, is there a space, do you have a hyphen, is it translated?** If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.

**We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.**

Once your re-verify your student status you will be able to re-enable Copilot.

**Reply by SammyWhamy** [reply]
Sorry for my rude reply. But please read everything I have said. My name is CORRECT on my profile AND on my documents. The documents however also list other names, I have already logged in and out a million times, I've submitted 12 applications and NOTHING. GOES. THROUGH.

Not to mention, it seems every single hour someone has the same issue as me, and nothing is getting solved. It's honestly ridiculous how you have told me to update my name 3 times now while I've literally stated I already did that in the original question.

Again, I need a HUMAN to look at my documents and manually verify that I am in fact still enrolled, I have plenty of proof that this is the case, I just simply cannot get past the automatic checks.

**Reply by SammyWhamy** [reply, marked as answer]
After trying another few times and using a virtual camera with an uploaded picture to bypass the no-upload requirement and get proper quality, it finally, suddenly worked

**Reply by ahmadhassan7** [reply]
Sorry to break in the conversation @samus-aran  can you please help me regarding my query.
I had submitted my documents about 6 days ago and I still got no response over my re-verification application. 
My account is going to expire in 4 days.

**Reply by samus-aran** [reply]
@SammyWhamy I appreciate your honesty and I don't think your answer was rude. I understand it can be frustrating when you believe you have done everything you can and still receive a failure. But let me assure you that what I explicitly said in my previous comments played a huge part in the verification and I am happy that you actually updated your profile to reflect that. I'm sure using the virtual camera helped but making sure your profile info corresponds to your evidence is key. Our verification system can be extremely robust and detects differences as minor as spaces 👀 
Aside from that I'm glad you were successful 🎉 What offer or part of Global Campus are you most excited to try first?

**Reply by samus-aran** [reply]
Hi @ahmadhassan7

Please open up a separate discussion so that we can look at your query there and not divert this discussion.
